src/sentry/migrations/0070_incident_snapshot_support.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.29 on 2020-04-28 21:29
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.utils.timezone
+
+
+class Migration(migrations.Migration):
+    # This flag is used to mark that a migration shouldn't be automatically run in
+    # production. We set this to True for operations that we think are risky and want
+    # someone from ops to run manually and monitor.
+    # General advice is that if in doubt, mark your migration as `is_dangerous`.
+    # Some things you should always mark as dangerous:
+    # - Large data migrations. Typically we want these to be run manually by ops so that
+    #   they can be monitored. Since data migrations will now hold a transaction open
+    #   this is even more important.
+    # - Adding columns to highly active tables, even ones that are NULL.
+    is_dangerous = False
+
+    # This flag is used to decide whether to run this migration in a transaction or not.
+    # By default we prefer to run in a transaction, but for migrations where you want
+    # to `CREATE INDEX CONCURRENTLY` this needs to be set to False. Typically you'll
+    # want to create an index concurrently when adding one to an existing table.
+    atomic = True
+
+
+    dependencies = [
+        ('sentry', '0069_remove_tracked_superusers'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='incidentsnapshot',
+            name='status',
+            field=models.PositiveSmallIntegerField(db_index=True, default=1),
+        ),
+        migrations.AddField(
+            model_name='incidentsnapshot',
+            name='target_run_date',
+            field=models.DateTimeField(default=django.utils.timezone.now),
+        ),
```

Comment:
Just noting that these defaults should be fine, since this table has ~2.5k rows.